.gitignore

```diff
@@ -4,3 +4,6 @@ conduit.dhall
 conduit-test.dhall
 conduit-server.cabal
 stack*.lock
+/.idea/
```

Comment:
@ishmum123, can you help resolve the latest merge conflicts

Reply:
@nodew can you please check now?